tests/intg/test_slide_gen_chain.py

```
@@ -0,0 +1,258 @@
+        client1 = MockOlmClient(responses=mock_responses)
+        client2 = MockOlmClient(responses=mock_responses)
+
+        chain1 = SlideGenChain(llm=client1)
+        chain2 = SlideGenChain(llm=client2)
+
+        # Verify they are independent instances
+        assert chain1.llm is not chain2.llm
+        assert chain1 is not chain2
+
+        # Both should work with the same mock responses
+        script_content = "Test script for concurrent usage"
```

Ruff (0.12.2)

Local variable `script_content` is assigned to but never used

Remove assignment to unused variable `script_content`

(F841)